_includes/manuals/1.10/4.3.6_smartproxy_puppet.md

```diff
@@ -124,6 +124,58 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+Here is a quick howto to enable puppetssh on your server.
+
+On the `foreman-proxy server`, enable the puppetssh provider and uncomment user and ssh keys settings:
+<pre>
+sed -i 's!^#\?\(:puppet_provider:\).*$!\1 puppetssh!' \
```

Comment:
It might be clearer in this part to show the finished config file, just so we don't assume too much about what's already in the config file.